<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plotter</class>
 <widget class="QDialog" name="Plotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ergebnisse </string>
  </property>
  <widget class="QCustomPlot" name="tempGraph" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="druckGraph" native="true">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>19</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="speedGraph" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>379</y>
     <width>911</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
